logstash/logstash.php

```diff
@@ -0,0 +1,22 @@
+<?php
+/**
+ * Plugin Name: VIP Logstash integration
+ * Description: Helper functions and classes for logstash integration.
+ * Author: Automattic
+ * License: GPL version 2 or later - http://www.gnu.org/licenses/old-licenses/gpl-2.0.html
+ */
+
+require_once __DIR__ . '/class-logger.php';
```

Comment:
I think we're missing `namespace Automattic\VIP\Logstash;` here?

Reply:
Added in 23d6080b108b26d6c052af4f5a2622f331ef2a84